styleguide/component.stories.js

```diff
@@ -53,7 +53,8 @@ ${usage || ''}`,
 
 // Storybook section setup
 export default {
-  title: '3: Components',
+  title: 'Components',
```

Comment:
This was the trick. Sets the title to the new name but keeps the `id` as the old one for the URLs.